support-lib/wasm/djinni_wasm.cpp

```diff
@@ -60,7 +60,13 @@ const em::val& JsProxyBase::_jsRef() const {
 
 void JsProxyBase::checkError(const em::val& v) {
     if (v.instanceof(em::val::global("Error"))) {
-        throw JsException(v);
+        auto cppExceptionPtr = v["cppExceptionPtr"];
```

Comment:
Small nit, but any of these fields we introduce internal to djinni, maybe we prefix them with _djinni_ or something? Might make it clearer for other code that touches these objects

Reply:
I already changed this to "_djinni_cpp_exception_ptr"

Reply:
nice, missed the last commit